src/test/JerryProtocolHandler.test.ts

```diff
@@ -15,6 +15,13 @@ function encodeArray(byte: number, str: string) {
   return array;
 }
 
+function addVersion(configArray) {
+  let version = new Uint8Array(4);
+  setUint32(true , version, 0, SP.JERRY_DEBUGGER_VERSION);
```

Comment:
The first parameter is the byte order to use, which is available from the configArray. Please use that value instead of this constant true.